ci-operator/step-registry/ibmcloud/provision/post/ibmcloud-provision-post-ref.yaml

```diff
@@ -0,0 +1,19 @@
+ref:
+  as: ibmcloud-provision-post
+  from_image:
+    namespace: ocp
+    name: "4.13"
+    tag: upi-installer
+  grace_period: 10m
+  commands: ibmcloud-provision-post-commands.sh
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  env:
+  - name: CREATE_CLUSTER_RESOURCE_GROUP 
```

Comment:
we already created this separate step, then we do not need this env any more, when you need this step in some install workflow, just insert this step into the installworkflow.